### DISCUSSION TITLE
Looping mode where windows can wrap off 1 side of the screen onto the other

### DISCUSSION THREAD

**Discussion by technobaboo** [YaLTeR/niri#174] [Ideas]
i know it sounds stupid but imagine you had a 360deg monitor all around you (totally not making that or anything) and you wanted to rotate your panels and all (similar to #45 but it'd be seamless)

**Comment by YaLTeR** [comment]
What if you spawn one more window and there's now not enough space?

**Reply by technobaboo** [reply]
i guess then it'd just clip off to the side like windows do now, like the carousel

**Reply by YaLTeR** [reply]
So what's the difference to the current behavior?

**Reply by technobaboo** [reply]
the difference would be that you could keep going right/left and the first/last window would be there... wait this is just the carousel proposal isn't it

sorry to make a new discussion for the same feature request

**Comment by jgregoire** [comment]
A variable-length ring buffer would be pretty cool. You could create looping slideshows just by opening a bunch of image windows and focusing right every X seconds via script.

**Reply by YaLTeR** [reply]
Lol that's a cool idea, but not sure it's in scope since it would probably require not very trivial changes to the code.
